lib/run-script-against-repositories.js

```diff
@@ -1,7 +1,11 @@
+function filterCancelledRoutesFromResults(results) {
+  return results.filter(Boolean);
+}
+
 export async function runScriptAgainstRepositories(octokit, repositories, script, options) {
-  for (const repository of repositories) {
+  const results = await Promise.all(repositories.map(async repository => {
```

Comment:
I think I'd prefer to run the scripts on repositories in sequential order, it makes the logs easier to read. And makes it easier to resume. And in general feels more resilient?

Reply:
my goal for switching to `.map` was for building the list of results while iterating through the list rather than mutating a separate map outside of the `for` loop. i hadnt considered that it also changed the sequential iteration to one that could be interwoven 🤔